src/deepsparse/v2/operators/engine_operator.py

```diff
@@ -87,7 +87,7 @@ def __init__(
         self._engine_args = engine_args
```

Comment:
we have engine_args, and engine_kwargs, but both are dicts; aren't the names slightly misleading?